ComponentLibrary/component-library.css

```diff
@@ -0,0 +1,289 @@
+/* Import */
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+/* Root */
+:root {
+    --primary-color: #ff6000;
+    --primary-dark: #282b2d;
+    --primary-grey: #93979c;
+    --light-grey: #c9d6df;
+    --white: #ffffff;
+    --red: #ff005c;
+    --green: #2eb872;
+    --yellow: #ffc300;
+    --blue: #0074e4;
+}
+
+/* Body */
+body {
+    font-family: Inter, sans-serif;
+}
+
+/* Avatar */
+.avatar {
+    border-radius: 100%;
+}
+.avatar-lg {
+    width: 80px;
+}
+.avatar-md{
+    width: 60px;
+}
+.avatar-sm {
+    width: 40px;
+}
+
+/* Alert */
+.alert {
+    border-radius: 5px;
+    padding: 1rem;
+    color: var(--white);
+    max-width: 60%;
+    margin: auto;
+}
+.danger-alert {
+    background-color: var(--red);
+}
+.success-alert {
+    background-color: var(--green);
+}
+.warning-alert {
+    background-color: var(--yellow);
+}
+.informative-alert {
+    background-color: var(--blue);
+}
+
+/* Badges */
+.badge {
+    color: var(--white);
+    background-color: var(--red);
+}
+.square-badge {
+    padding: 5px;
+}
+.circle-badge {
+    border-radius: 100%;
+    padding: 1px 5px;
+}
+.oval-badge {
```

Comment:
You could name them something like badge-[componentSpecific]. Like badge-oval, badge-circle. But it's totally subjective and optional. It gives more context that's it. Nothing wrong and right.

Reply:
Will be doing that.